x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/missing_privileges_callout.cy.ts

```diff
@@ -24,8 +24,8 @@ import {
 import { ruleDetailsUrl } from '../../../urls/rule_details';
 
 const loadPageAsReadOnlyUser = (url: string) => {
-  login(ROLES.reader);
-  visit(url, { role: ROLES.reader });
+  login(ROLES.t1_analyst);
```

Comment:
Can we actually use the `Roles.Viewer` here? Security provides the default roles as well which I don't think are accounted for in this PR, but are available in the project roles `[none, admin, editor, viewer]`. I think viewer would be more accurate here than `t1_analyst`.

Reply:
Or, change the name of the function to `loadPageAsT1Analyst` to be more accurate. We should probably create utils for the other roles as well to keep things dry, but that can be in a follow up PR? Thanks for doing this work, it's awesome!

Reply:
@michaelolo24 thank you for the review 🙏  I renamed the function [here](https://github.com/elastic/kibana/pull/169017/commits/37c2403c1f30864cfe5764ae45412d0db25f2802).

Reply:
Hey @maximpn, no thank you for doing this, it's really helpful! QQ, I'm not sure if there was a rebase issue, but I don't see the change?